jenkins/pipelines/node-linter.jenkinsfile

```diff
@@ -42,15 +42,22 @@ pipeline {
             steps {
                 // Calling with `returnStatus` suppresses automatic failures
                 sh(script: "${env.MAKE} lint-md-build", returnStatus: true)
+                sh(script: "${env.MAKE} lint-py-build", returnStatus: true)
             }
         }
          
         stage('Run tests') {
             steps {
                 script {
+                    def node_version = sh(script: "@python  tools/getnodeversion.py", returnStdout: true)
+                    def lint_py3_ret = 0
+                    if (node_version > '11') {
```

Comment:
Is this a safe comparison?

Reply:
Tweaked just enough to get it to work with `8.`
```Groovy
node_version > "11." && node_version.take(1) != '8'
```

Reply:
P.S. I don't think I like groovy